watson-developer-cloud/discovery-components
# chore: Update carbon-components-react to at least 7.7.0

#### What do these changes do/fix?
Upgrade `carbon-components-react` to at least 7.7.0 so that we no longer see `console.error ../../node_modules/prop-types/factoryWithTypeCheckers.js:23
      Warning: Invalid arguments supplied to oneOf, expected an array, got 4 arguments. A common mistake is to write oneOf(x, y, z) instead of oneOf([x, y, z]).` when running our tests.

#### How do you test/verify these changes?
Run `yarn test` and see that this error is no longer there. Also check that everything else still works as expected (build, storybook, etc.)

#### Have you documented your changes (if necessary)?
n/a

#### Are there any breaking changes included in this pull request?
No
